app/repositories/app_event_repository.rb

```diff
@@ -15,6 +15,21 @@ class AppEventRepository
                              docker_credentials].freeze
       SYSTEM_ACTOR_HASH = { guid: 'system', type: 'system', name: 'system', user_name: 'system' }.freeze
 
+      def create_app_readiness_changed_event(app, readiness_changed_payload)
```

Comment:
hey @ameowlia, where is this function being called, i couldn't find it?

Reply:
My guess is that I forgot to delete that code. I initially (erroneously) added an app event instead of an app.process event. It's probably part of the app event that I should've deleted. I'll test to make sure everything still works if I remove it tomorrow. Good catch!

Reply:
I removed the code. Units still pass. And when I deployed it everything still works.